requirements.txt

```diff
@@ -14,6 +14,7 @@ django-toolbelt==0.0.1
 django-treebeard==4.0.1
 djangorestframework==3.4.6
 docutils==0.12
+elasticsearch==1.2.0
```

Comment:
the README is pretty out of date in general. I don't think we'll use heroku as a permanent solution, so don't worry about heroku-specific things much until we know the deploy better.